components/infobox/commons/infobox_person.lua

```diff
@@ -247,6 +246,8 @@ function Person:createInfobox()
 		)
 	end
 
+	self:_definePageVariables(args)
```

Comment:
I would consider running it before the setLpdbData still. They could be some dependency still

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/5430#discussion_r1947757798